### ISSUE TITLE
Settings: Change disabled primary buttons to secondary buttons

### ISSUE THREAD

**Issue by jeffgolenski** [Automattic/jetpack#19136]
Scattered throughout our settings within the plugin we have a bunch of disabled primary buttons that read "save settings." These buttons are disabled until the settings of their respective cards are changed, and then they become active for the customer to save the settings.

To clean things up, I'm proposing that we remove `is-primary` until the buttons become enabled. They have the potential to distract customers and shouldn't be primary until they are active.

### Current settings areas
*(Performance and Sharing areas are fine as-is)*

![Current settings](https://user-images.githubusercontent.com/214813/110844693-b982ac80-8277-11eb-8e73-525af72dbc7c.jpg)

### Proposed changes to save settings buttons

![functionality](https://user-images.githubusercontent.com/214813/110844785-d61ee480-8277-11eb-8708-a7825d966a7c.jpg)

### Diagram of all targeted buttons
![proposed changes](https://user-images.githubusercontent.com/214813/110844722-c1425100-8277-11eb-9280-a9f2dbb91899.jpg)

On the surface it seems fairly simple. Remove `is-primary` from all the `disabled` buttons. When `disabled` is removed, we should apply `is-primary`. Thoughts?

Also, under Writing > Composing there appears to be a save button in an area where only toggles exist. If that's the case, I think we can safely remove it from that particular card.

**Comment by folletto** [comment]
I'd rather update the disabled style of `is-primary`. 
It feels quite a contradiction to change the _type_ of button when the state change. If that's the case, and we feel strongly about it, I think it would point of some issues with the `is-primary` + `disabled` state.

**Comment by jeffgolenski** [comment]
@folletto Sure that makes sense. I initially thought perhaps that was the default core style for disabled primaries and that's why I suggested a change of button type, rather than overriding core styles. But at second glance of the code, it does indeed look like that's a Jetpack style!

The goal of the issue remains the same, regardless of which method we move forward with. Reducing that visual clutter!

**Comment by folletto** [comment]
Yes, and just to clarify: we need to be careful as it's a design language (either ours or WP Admin's), so we either manage to change it at its root, or we shouldn't introduce exceptions. If it was WP Admin's, then I'd probably suggest to not proceed with this change at all, and bring the discussion to WordPress Core instead.

**Comment by keoshi** [comment]
For reference, this is what the latest version of [Core styles look like in Figma](https://www.figma.com/file/e4tLacmlPuZV47l7901FEs/WordPress-Design-Library?node-id=7%3A42&viewport=1580%2C-2424%2C1):

![image](https://user-images.githubusercontent.com/390760/110944544-818d6f00-8334-11eb-9b30-e85f0735dccd.png)

Other than a subtle tonal differences in the Primary button, the effect is the same. Would it make sense to make all of these Secondary buttons to solve this issue? To me that button carries much more meaning when swapping from a disabled state to the regular one.

**Comment by folletto** [comment]
Yes that's an option. We could also consider to NOT show a button at all until there's a change (as long as we don't create a layout adjustment).

**Comment by keoshi** [comment]
Love that, as we can still use the Primary button in that case _AND_ get a cleaner layout.

**Comment by jeffgolenski** [comment]
That was my initial thought as well, but do you think that jump is alright for the user? If so, perhaps that's the the best option

**Comment by folletto** [comment]
The issue imho there is that often when that is done there's a reflow of the overall UI (adding space, showing a new bar, etc). If we make sure there's never a layout change when the button appears, I feel it can work. At least, worth a test. :) 

In the long term, I feel the overall page could get some rethinking with the above issue in mind too, but for now, this seems a good thing to try.

**Comment by folletto** [comment]
The issue imho there is that often when that is done there's a reflow of the overall UI (adding space, showing a new bar, etc). If we make sure there's never a layout change when the button appears, I feel it can work. At least, worth a test. :) 

In the long term, I feel the overall page could get some rethinking with the above issue in mind too, but for now, this seems a good thing to try.

**Comment by jeffgolenski** [comment]
That was my initial thought as well, but do you think that jump is alright for the user? If so, perhaps that's the the best option.
